source/core/src/main/com/csse3200/game/GdxGame.java

```diff
@@ -161,10 +160,6 @@ public void dispose() {
         getScreen().dispose();
     }
 
-    private Screen newScreen(ScreenType screenType, Screen screen, ServiceContainer container) {
-        return newScreen(screenType, screen, container, null, null);
-    }
-
```

Comment:
To anyone reading, this isn't called, but if it were it would just recurse until stack overflow and crash the game.

Reply:
Why would it recurse more than once? It calls the newScreen with 5 arguments which doesn't call newScreen again.

Reply:
Ah, I misread the function - didn't realise there was another constructor with more arguments - I thought on first glance it was calling itself. Regardless it's not used and just calls the method with the same name - someone can put it back if it seems necessary but as of right now it seems redundant.